packages/renderer/src/lib/kube/KubeContextUI.ts

```diff
@@ -44,3 +44,13 @@ export function setKubeUIContextError(contexts: Context[], contextName: string,
 export function clearKubeUIContextErrors(contexts: Context[]): Context[] {
   return contexts.map(ctx => ({ ...ctx, error: undefined }));
 }
+
+export function clearKubeUIContextError(contexts: Context[], contextName: string): Context[] {
```

Comment:
question: wondering if it can't be merged with the previous function

ie: If I provide a contextName parameter then it'll reset only for a given context

Reply:
ok, I merged both functions

Reply:
ok thx